pkg/eks/api/api.go

```diff
@@ -30,9 +33,20 @@ const (
 var SupportedRegions = []string{
 	EKSRegionUSWest2,
 	EKSRegionUSEast1,
+	EKSRegionUSEast2,
 	EKSRegionEUWest1,
 }
 
+// IsSupportedRegion returns true if EKS is available in the region
+func IsSupportedRegion(region string) bool {
```

Comment:
We have this already: https://github.com/weaveworks/eksctl/blob/0a87749aca78b5a94541651eb95bc4b41e4f2914/pkg/eks/api.go#L125-L133